<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewStaggering</class>
 <widget class="QDialog" name="NewStaggering">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Nouvel echelonnement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="studentName_3">
         <property name="text">
          <string>Niveau</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="levelChoice">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>L1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>M2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="studentName_2">
         <property name="text">
          <string>Parcours</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="categoryChoice">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Pro</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ig</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gb</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="studentName">
       <property name="text">
        <string>Nom de l'etudiant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="studentCombobox">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_60">
       <property name="text">
        <string>Somme totale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="totalAmount">
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date de fin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dueDate"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="saveStaggeringbtn">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveStaggeringbtn</sender>
   <signal>accepted()</signal>
   <receiver>NewStaggering</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveStaggeringbtn</sender>
   <signal>rejected()</signal>
   <receiver>NewStaggering</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
